License.txt

```diff
@@ -1,39 +1,29 @@
-Copyright (c) 2007. Clarius Consulting, Manas Technology Solutions, InSTEDD
-http://www.moqthis.com/
+BSD 3-Clause License
+
+Copyright (c) 2007, Clarius Consulting, Manas Technology Solutions, InSTEDD
 All rights reserved.
 
-Redistribution and use in source and binary forms, 
-with or without modification, are permitted provided 
-that the following conditions are met:
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright notice,
+    this list of conditions and the following disclaimer.
 
-    * Redistributions of source code must retain the 
-    above copyright notice, this list of conditions and 
-    the following disclaimer.
-    
-    * Redistributions in binary form must reproduce 
-    the above copyright notice, this list of conditions 
-    and the following disclaimer in the documentation 
-    and/or other materials provided with the distribution.
-    
-    * Neither the name of Clarius Consulting, Manas Technology Solutions or InSTEDD nor the 
```

Comment:
`the name of Clarius Consulting, Manas Technology Solutions or InSTEDD` &rarr; `the names of the copyright holders`